include/ddc/kernels/fft.hpp

```diff
@@ -602,7 +714,23 @@ typename DDimFx::template Impl<DDimFx, Kokkos::HostSpace> init_fourier_space(
     return std::move(impl);
 }
 
-// FourierMesh, first element corresponds to mode 0
+/**
+ * @brief Get the Fourier mesh.
+ *
+ * Compute the Fourier (or spectral) mesh on which the Discrete Fourier Transform of a
+ * spatial discrete function is defined.
+ *
+ * The uid identifies the mode (ie. ddc::DiscreteElement<DDimFx>(0) corresponds to mode 0).
```

Comment:
```suggestion
```

We cannot suggest the user to build a `DiscreteElement` over a discrete dimension that defines attributes.

Reply:
can you explain why ? I don't understand this limitation

Reply:
Because if we expose the uid, users will start relying on it and making implicit assumptions in the source code. We want to avoid this.